apps/ci/ci-pending.sh

```diff
@@ -31,5 +31,16 @@ for i in `find data/sql/updates/pending* -name "*.sql" -type f`; do
     fi
 done
 
+for i in `find data/sql/updates/pending* -name "*sql" -type f`; do
+    if $(cat "$i"|sed "s/'.*'\(.*\)/\1/g"|grep -q -i -E "broadcast_text"); then
+        echo "> broadcast_text check - OK"
+    else
```

Comment:
could someone just proof check that I have the if/else in the right order and that the regex is correct? :D

Reply:
just add test file

Reply:
True. Didnt think of that hahaha